DoxygenRule.cmake

```diff
@@ -85,8 +85,8 @@ if(NOT DOXYGEN_CONFIG_FILE)
 endif()
 
 add_custom_target(doxygen_install_${PROJECT_NAME}
-  ${CMAKE_COMMAND} -DCMAKE_INSTALL_LOCAL_ONLY=ON
-  -P ${PROJECT_BINARY_DIR}/cmake_install.cmake
+  ${CMAKE_COMMAND}
+  -P ${PROJECT_BINARY_DIR}/${PROJECT_INCLUDE_NAME}/cmake_install.cmake
```

Comment:
At least in Equalizer the examples are also doxygen-ed. INSTALL_DEPENDS should include applications imo. That used to work at least.